datahub-web-react/src/app/settings/posts/ManagePosts.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from 'antd';
+import React from 'react';
+import styled from 'styled-components/macro';
+import { PostList } from './PostsList';
+
+const PageContainer = styled.div`
+    padding-top: 20px;
+    width: 100%;
+    height: 100%;
+`;
+
+const PageHeaderContainer = styled.div`
+    && {
+        padding-left: 24px;
+    }
+`;
+
+const PageTitle = styled(Typography.Title)`
+    && {
+        margin-bottom: 12px;
+    }
+`;
+
+const ListContainer = styled.div``;
+
+export default function ManagePosts() {
+    return (
+        <PageContainer>
+            <PageHeaderContainer>
+                <PageTitle level={3}>Home Page Posts</PageTitle>
+                <Typography.Paragraph type="secondary">View and manage your DataHub Posts.</Typography.Paragraph>
```

Comment:
Maybe add something about the posts appearing on the landing page.

"View and manage the pinned posts that appear to users on the landing page. "

Reply:
oh yeah so much better